java-checks/src/test/files/checks/ReuseRandomCheck.java

```diff
@@ -0,0 +1,39 @@
+import java.util.Random;
+
+public class A {
+
+  static Random s = new Random(); // Compliant for static field
+  Random f = new Random(); // Compliant for field
+
+  A() {
+    Random v = new Random(); // Compliant in constructor
+  }
+
+  void func(long seed, Random p) {
```

Comment:
could you please name your variable in your testcase in a way that it is understandble : field could be named field, s -> staticField, p -> param, and a b or c localVar1, localVar2, etc.